### PR TITLE
Create preliminary validation parameters for quidel_covidtest

### PR DESCRIPTION
**Pull request by sgsmob**
### Description
Add a preliminary set of validation parameters for `quidel_covidtest`

### Fixes 
- Partially addresses #725

### CODE REVIEW

File: quidel_covidtest/params.json.template [hunk 1]
```diff
@@ -17,5 +17,21 @@
...
+    },
+    "static": {
+      "minimum_sample_size": 50
+    },
+    "dynamic": {
+      "ref_window_size": 7,
+      "smoothed_signals": [
+        "covid_ag_smoothed_pct_positive"
+      ]
+    }
```
Comment (chinandrew) [inline quidel_covidtest/params.json.template:36]: think you're missing a closing curly bracket here

**Review by chinandrew — APPROVED** [review]
:shipit::shipit::shipit:

**Comment by sgsmob** [comment]
Please wait on #943 to merge.

File: ansible/templates/quidel_covidtest-params-prod.json.j2 [hunk 2]
```diff
@@ -17,5 +17,22 @@
     "bucket_name": "{{ quidel_aws_bucket_name }}",
     "wip_signal": [""],
     "test_mode": false
+  },
+  "validation": {
+    "common": {
+      "data_source": "quidel",
+      "span_length": 14,
+      "end_date": "today",
```
Comment (krivard) [inline ansible/templates/quidel_covidtest-params-prod.json.j2:25]:
```suggestion
      "end_date": "today-5",
```

File: quidel_covidtest/params.json.template [hunk 3]
```diff
@@ -17,5 +17,22 @@
     "bucket_name": "",
     "wip_signal": [""],
     "test_mode": false
+  },
+  "validation": {
+    "common": {
+      "data_source": "quidel",
+      "span_length": 14,
+      "end_date": "today",
```
Comment (krivard) [inline quidel_covidtest/params.json.template:25]:
```suggestion
      "end_date": "today-5",
```

**Review by krivard — COMMENTED** [review]
Updates to end_date referencing min_lag from meta

**Pull request merged**